daml/Marketplace/Distribution/Auctioneer.daml

```diff
@@ -0,0 +1,58 @@
+module Marketplace.Distribution.Auctioneer where
```

Comment:
I'd call this the `Distributor` role, it can be used later to spawn other types of distribution.

Reply:
Done